src/Dialect/Krnl/Krnl.td

```diff
@@ -89,13 +89,22 @@ def KrnlCallOp : Op<Krnl_Dialect, "call",
       DefaultValuedAttr<SI64Attr, "1">:$numOfOutput,
       Variadic<AnyType>:$parameters);
 
+  // Return Value for the Call.
+  // No return if the type is NoneType (void in llvm)
+  // Only scalar type is supported now.
+  // In future, return of memref can be supported with pointer of OMTensor.
+  // The returned memref will be created inside the call. 
+  let results = (outs Variadic<AnyTypeOf<[AnyFloat, AnyInteger]>>:$returnValue);
```

Comment:
Having a Variadic for zero or one feels a bit inelegant, but it woks.

Reply:
Is there a way to define the maximum number of variadic?

Reply:
There are constraints that can be added on values, not sure about the variadic